Header header

uint64 index_to_replace
Waypoint replacement_waypoint